src/cswinrt/strings/additions/Windows.Storage.Streams/WindowsRuntimeBufferExtensions.cs

```diff
@@ -140,6 +156,25 @@ public static byte[] ToArray(this IBuffer source, uint sourceIndex, int count)
 #endregion (IBuffer).ToArray extensions for copying to a new (Byte [])
 
 
+#region (IBuffer).AsMemory extensions for copying to a new (Byte [])
+
+        public static Memory<byte> AsMemory(this IBuffer source)
```

Comment:
This is deceivingly unsafe. The GC could collect the underlying `IBuffer` while someon's using the span.

Reply:
I don't follow. If the IBuffer is backed by a Memory<byte> that should be safe. If not, the Memory instance should keep the IBufferMemoryManager alive which is keeping the IBuffer alive?

Reply:
Not sure why you resolved this, I didn't get a chance to reply. This is not safe because of this:
```csharp
IBuffer buffer = GetSomeBuffer();
Span<byte> span = buffer.AsMemory().Span;
// <-- GC runs here
DoStuff(span); // Boom 💥
```

Reply:
Doesn't that equally apply to string?

```csharp
string buffer = GetSomeString();
Span<char> span = buffer.AsMemory().Span;
// <-- GC runs here
DoStuff(span); // Boom 💥

Reply:
I resolved as we discussed offline, but maybe there was some confusion around what I meant :). Either way we can continue the conversation outside this comment.